example_code/shmem_put_signal_example.c

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <shmem.h>
+
+#define MAX_SIZE (2<<10)
+#define VAL_USED 10
+
+int
+main(int argc, char* argv[])
+{
+    int i, err_count  = 0;
+
+    shmem_init();
+
+    size_t    size    = MAX_SIZE;
+    int       me      = shmem_my_pe();
+    int       n       = shmem_n_pes();
+    int       pe      = (me + 1)%n;
+
+    uint64_t* message = malloc(size * sizeof(uint64_t));
+    uint64_t* data    = shmem_malloc(size * sizeof(uint64_t));
+    uint64_t* signals = shmem_malloc(sizeof(uint64_t));
```

Comment:
Feedback from July 23 reading: Might it better emphasize the symmetric heap or static variable distinction to use a static variable here?

Reply:
resolved - closing outdated changes.
Example program went through multiple changes and the comments are resolved.